src/mrinufft/operators/base.py

```diff
@@ -50,7 +50,12 @@ def list_backends(available_only=False):
 
 
 def get_operator(
-    backend_name: str, wrt_data: bool = False, wrt_traj: bool = False, *args, **kwargs
+    backend_name: str,
+    wrt_data: bool = False,
+    wrt_traj: bool = False,
+    paired_batch_size: int = None,
```

Comment:
If I understand correctly `paired_batch_size` is used only if we want to process for different smaps right? 

Can we rename in that case? `smap_batch_size` ? @paquiteau opinions?

Reply:
From my understanding in this "paired" mode, the batch size of smaps and data is the same. Therefore I don't think we need an extra integer, a simple boolean flag `paired_batch` (or `paired_batch_mode` fwiw) should be enough ?